logging/telemetry.go

```diff
@@ -264,6 +266,7 @@ func (t *telemetryState) logTelemetry(l logger, message string, details interfac
 	entry := l.entry.WithFields(Fields{
 		"session":      l.GetTelemetrySession(),
 		"instanceName": l.GetInstanceName(),
+		"v":            l.GetTelemetryVersion(),
```

Comment:
Could you use a full string for these?
```suggestion
		"version":            l.GetTelemetryVersion(),
```

Reply:
The concern was that it would use up more bandwidth, so squish it down to the minimum.